src/library/dlgtrackinfo.cpp

```diff
@@ -744,7 +744,7 @@ mixxx::UpdateResult DlgTrackInfo::updateKeyText() {
 }
 
 void DlgTrackInfo::displayKeyText() {
-    const QString keyText = KeyUtils::keyToString(m_trackRecord.getKeys().getGlobalKey());
+    const QString keyText = m_pLoadedTrack ? m_pLoadedTrack->getKeyText() : QStringLiteral("");
```

Comment:
Hm, I wonder if this is enough. In #14008 I found that getKeyText() could be empty and I had to use getKey().

Reply:
yeah, might also be `getKeys().getGlobalKey()`.

I'm still digging a bit, trying to understand how the metadata flow is supposed to work.